src/renault_api/exceptions.py

```diff
@@ -0,0 +1,7 @@
+"""Exceptions for Renault API."""
+
+
+class RenaultError(Exception):
```

Comment:
`RenaultError` --> `RenaultException`

Error in Python usually means something than we should not catch because it was fatal for the program.
Exceptions on the other hand are made to be handled.